### DISCUSSION TITLE
Cant access btc site i get security warning

### DISCUSSION THREAD

**Discussion by ItchyPong** [btcpayserver/btcpayserver#2238] [Technical Support] [answered]
Your connection is not private
Attackers might be trying to steal your information from btcpay.vitalherbs.co.uk (for example, passwords, messages or credit cards). Learn more
NET::ERR_CERT_AUTHORITY_INVALID

same or slightly diff on all browswers?

**Comment by pavlenex** [comment, marked as answer]
Try `btcpay-down.sh` then `btcpay-up.sh` for some reason SSL certificate wasn't issued, you may want to check it's logs to see why. Best is to come to our chat.btcpayserver.org so someone can help you through the process.
